modules/F50_base_aggregator.sh

```diff
@@ -96,47 +96,54 @@ output_details() {
   local DATA=0
   if [[ "$FILE_ARR_COUNT" -gt 0 ]]; then
     print_output "[+] ""$ORANGE""$FILE_ARR_COUNT""$GREEN"" files and ""$ORANGE""$DETECTED_DIR"" ""$GREEN""directories detected."
+    write_link "s05"
     echo "files;\"$FILE_ARR_COUNT\"" >> "$CSV_LOG_FILE"
     echo "directories;\"$DETECTED_DIR\"" >> "$CSV_LOG_FILE"
     DATA=1
   fi
   ENTROPY_PIC=$(find "$LOG_DIR" -xdev -type f -iname "*_entropy.png" 2> /dev/null)
   if [[ -n "$ENTROPY" ]]; then
     print_output "[+] Entropy analysis of binary firmware is:""$ORANGE""$ENTROPY"
-    echo "entropy_value;\"$ENTROPY\"" >> "$CSV_LOG_FILE"
```

Comment:
why removing this?

Reply:
Oh, it shouldn't be removed. I think it was a mistake while merging Master into my branch. I will fix this.